cluster-provision/gocli/cmd/run.go

```diff
@@ -79,10 +79,17 @@ func NewRunCommand() *cobra.Command {
 	run.Flags().Uint("ocp-port", 0, "port on localhost for the ocp cluster")
 	run.Flags().Uint("k8s-port", 0, "port on localhost for the k8s cluster")
 	run.Flags().Uint("ssh-port", 0, "port on localhost for ssh server")
+	run.Flags().Uint("prometheus-port", 0, "port on localhost for prometheus server")
+	run.Flags().Uint("grafana-port", 0, "port on localhost for grafana server")
 	run.Flags().String("nfs-data", "", "path to data which should be exposed via nfs to the nodes")
 	run.Flags().String("log-to-dir", "", "enables aggregated cluster logging to the folder")
 	run.Flags().Bool("enable-ceph", false, "enables dynamic storage provisioning using Ceph")
 	run.Flags().Bool("enable-istio", false, "deploys Istio service mesh")
+	run.Flags().Bool("enable-prometheus", false, "deploys Prometheus operator")
+	run.Flags().UintP("prometheus-replicas", "x", 1, "number of prometheus replicas to start")
```

Comment:
ok, I can remove that.
But the motivation to have this option is for a large-scale scenario for scalability tests. It might be that in a scenario with 1k nodes,  a single prometheus point can become a bottleneck. I don't know if a single instance can sustain such a scenario...

Reply:
Realistically we can't create more than a handful of nodes. More than 100 or 200 is not realistic. `kubevirtci` is bound to a single underlying machine. Further I think that you would need sharding for that. As far as I understand kube-prometheus, this would only result in a more highly-available setup because all replicas would scrape the same targets.

Reply:
Oh, just ensure that you just replace all `replicas` occurences with `1` when applying the manifests. We really don't need any redundancy and it just consumes resources.

Reply:
It makes sense thanks!
Actually, I have already decreased all the replicas from 2 to 1. But I will remove the option to increase it